.github/workflows/podPublish.yml

```diff
@@ -1,12 +1,15 @@
 name: Pod-Publish
 on:
   push:
+    branches:
+      - main
```

Comment:
@harrieshin it seems like adding the branches here triggered a pod publish for any merge that happens into main:

https://github.com/microsoft/fluentui-apple/actions/runs/1106904940
https://github.com/microsoft/fluentui-apple/actions/runs/1117722702

Maybe we can use the release to trigger it instead?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onevent_nametypes
```
on:
  release:
    # Only use the types keyword to narrow down the activity types that will trigger your workflow.
    types: [published, created, edited]
```

Reply:
ah i think i missed the quotation to map it to refs/heads/main instead